iree/hal/vulkan/CMakeLists.txt

```diff
@@ -475,28 +475,21 @@ iree_cc_library(
     "vma_buffer.cc"
   COPTS
     "-DVK_NO_PROTOTYPES"
-    # Only needed in the implementation cc and not by external users.
-    "-DVMA_STATIC_VULKAN_FUNCTIONS=0"
-    # Silence some warnings.
-    "-Wno-nullability-completeness"
```

Comment:
It should hopefully not be needed with the system header <> include style - and if it is we can ignore it in code. This particular copt here broke MSVC builds, which is a good example of why we need to avoid these things :)